### PR TITLE
Issue 16

### PR DESCRIPTION
**Pull request by dreagle95**
Changed the format to a table #16

### CODE REVIEW

**Comment by dreagle95** [comment]
@raghunat I have tried to change it into a table, I'm not sure if I've used the proper format or not  yet

File: spots-api/package.json [hunk 1]
```diff
@@ -1,29 +0,0 @@
-{
```
Comment (stephenthedev) [inline spots-api/package.json]: Why did you delete this?

**Review by stephenthedev — DISMISSED** [review]

File: spots-api/package.json (hunk 1 above)
Comment (dreagle95) [reply spots-api/package.json]: That is a great question.  It certainly shouldn't have been deleted
